runtime/dart_controller.cc

```diff
@@ -129,8 +129,9 @@ tonic::DartErrorHandleType DartController::RunFromSource(
   Dart_Handle result = loader.LoadScript(main);
   LogIfError(result);
```

Comment:
I think we may need a check of result here as loader.LoadScript calls Dart_LoadScript which does a top level compile
of the loaded script.